contact-us/index.md

```diff
@@ -0,0 +1,83 @@
+---
+altLangPage: "contactez-nous.html"
+breadcrumbs: false
+dateModified: 2023-04-12
+description: "Contact the Digital Transformation Office about the Canada.ca design system."
+lang: en
+layout: default
+share: false
+nositesearch: true
+nomenu: true
+showFeedback: false
+noReportProblem: true
+title: "Contact the Digital Transformation Office"
+---
+Contact us to ask design questions or make suggestions about the design of Canada.ca.
+
+<div class="row">
+  <div class="col-md-8">
+    <details class="mrgn-tp-lg">
+      <summary>Information about Privacy</summary>
+      <p class="mrgn-tp-lg">We collect the personal information you submit through this contact form under the authority of the <i>Financial Administration Act</i>, so we can respond to you. Providing the information requested on this form is voluntary. The personal information collected will be available to those program staff responsible for administering the website, and will be used and protected in accordance with the <i>Privacy Act</i> and as described in Personal Information Bank PSU 914 Public Communications. Under the <i>Privacy Act</i>, you have the right to the correction of, access to, and protection of your personal information. If you have any questions about this privacy statement, contact the TBS Access to Information and Privacy Coordinator. If you are not satisfied with TBS’s response to your privacy concern, you may wish to contact the Office of the Privacy Commissioner.</p>
```

Comment:
should we change the i tags to cite tags? When we made the style guide change to allow italics for the names of acts, it was based on the understanding that they are supposed to be coded as citations, and the cite tag renders in italics. Can we check if these should be cite instead of i?

Reply:
Absolutely